src/VASSAL/build/module/GlobalOptions.java

```diff
@@ -167,6 +171,16 @@ public void propertyChange(PropertyChangeEvent e) {
 
       prefs.addOption(bug10295Conf);
     }
+    
+    // Move Fixed Distance trait (Translate) has been substantially re-written.
+    // Use new version by default. User may over-ride to use old buggy behaviour.
+    classicMfd = new BooleanConfigurer(
```

Comment:
I'm not sure there is a good answer. 

The usual way will be to flail about blindly until accidentally stumbling on an obscure comment on the forum, or noticing the option in the preferences and trying it.

If the User Guide where ever to updgraded from the current 3.1 version, that would be a good place.

I think moving the bug-related preferences to a separate 'Compatibility' tab would help highlight the options available for changing behaviour.

The current MFD behaviour is so broken that we really need this preference turned off by default. In my estimation, even modules that are already using MFD will work at least as well as they do now.

However, the users who need it turned on will probably not know that they need it turned on unless there is a big warning on the Module download page that they need to turn this option on.

It would be handy for a Module Developer to be able to set the default value for particular Module preferences that will be used when a module is first used on a system. At the moment, module preferences are set to the default that Vassal specifies, not what the module requires.

The 'Auto-report moves' is another option that would benefit from this. Some module handles all reporting internally by the pieces and need this option off. Most need it on.

Reply:
Ok. Maybe open a ticket for making that Compatiblity tab? And a second one for letting modules set those things?

(Incidentally, I'm not sure that the DirectX D3D setting that you would move to a Compatibility tab is even needed anymore. I haven't heard of anyone using it in ages.)